test/functional/test_framework/address.py

```diff
@@ -41,14 +41,6 @@ def scripthash_to_p2sh(hash, main = False):
     version = 16 if main else 19
     return byte_to_base58(hash, version)
 
-def key_to_p2pkh(key, main = False):
-    key = check_key(key)
-    return keyhash_to_p2pkh(hash160(key), main)
-
-def script_to_p2sh(script, main = False):
-    script = check_script(script)
-    return scripthash_to_p2sh(hash160(script), main)
-
```

Comment:
14519: this should be reverted

Reply:
@UdjinM6  Could you please confirm whether issues at https://gitlab.com/dashpay/dash/-/jobs/1508393724 need to be considered(any errors on running python linter). That is the reason I have removed some methods. Please confirm

Reply:
Linter fails there because 1. you broke `test/lint/lint-python-dead-code.sh` (see https://github.com/dashpay/dash/pull/4329#discussion_r690690560 below) and 2. there is an issue fixed by #4344